packages/studio/src/components/Viewport/ViewportButton.tsx

```diff
@@ -0,0 +1,29 @@
+import { useCallback, useRef, useState } from "react";
+import useRootClose from "@restart/ui/useRootClose";
+import ViewportMenu from "./ViewportMenu";
+
+export default function ViewportButton(): JSX.Element {
+  const [isOpen, setIsOpen] = useState(false);
+  const containerRef = useRef<HTMLDivElement>(null);
+  const icon =
+    "https://assets.brandfolder.com/q4zcv4-5x9x3c-cwpfeb/v/9458449/original/DevicesLaptopPhoneTablet_B.png";
```

Comment:
Is there an svg version of this image that we can commit to the repo? Then we could directly import the svg as a react component (we do this in a few places like `import { ReactComponent as Globe } from "../icons/globe.svg";`)

Reply:
(Or we could convert the png to an svg if there isn't a similar thing for png files)